typescript/packages/well-log-viewer/src/components/WellLogView.tsx

```diff
@@ -1204,6 +1217,7 @@ export function shouldUpdateWellLogView(
     // so compare only known key:values
     if (props.horizontal !== nextProps.horizontal) return true;
     if (props.welllog !== nextProps.welllog) return true;
+    if (props.wellLogSets !== nextProps.wellLogSets) return true;
```

Comment:
I was expecting something like:
if (props.wellLogSets ?? props.welllog  !== nextProps.wellLogSets ?? nextProps.welllog) return true;

This emphasis that wellLogSets  supersedes welllog

Reply:
I think the code would be harder to read if I cram it inside the same if-statement. But, I can reorder the two checks to match others, if wanted

Reply:
fine for me